pyscf/scf/addons.py

```diff
@@ -26,12 +26,16 @@
 from pyscf.gto import mole
 from pyscf.lib import logger
 from pyscf.scf import hf
+from pyscf.pbc.scf import addons as pbcaddons
```

Comment:
define smearing function and make a lazy import inside the function. PBC module has complicated module dependencies. import at scf module level may break "import scf"